state/storage.go

```diff
@@ -46,8 +46,9 @@ type Storage struct {
 func NewStorage(postgresURI string) *Storage {
 	db, err := sqlx.Open("postgres", postgresURI)
 	if err != nil {
-		logger.Panic().Err(err).Str("uri", postgresURI).Msg("failed to open SQL DB")
 		sentry.CaptureException(err)
+		// TODO: if we panic(), will sentry have a chance to flush the event?
+		logger.Panic().Err(err).Str("uri", postgresURI).Msg("failed to open SQL DB")
```

Comment:
Here and below: logger.Panic()/.Fatal() will blow up the current goroutine/program, so for sentry to have a chance at doing anything it needs to report before the panic/os.Exit call.

Don't think this is robust, but it gives sentry a chance to report the error.